newrelic.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+	/**
+	 * Array of application names.
+	 */
+	app_name: ['YOUR_PROJECT_NAME'],
```

Comment:
Why is this an array? I'm just curious :d

Reply:
Not sure, this whole file was generated by newrelic